src/Compilers/CSharp/Portable/Symbols/Metadata/PE/PEMethodSymbol.cs

```diff
@@ -928,13 +928,7 @@ public override bool IsExtensionMethod
                 // attributes are requested.
                 if (!_packedFlags.IsExtensionMethodIsPopulated)
                 {
-                    bool isExtensionMethod = false;
-                    if (this.MethodKind == MethodKind.Ordinary && IsValidExtensionMethodSignature()
-                        && this.ContainingType.MightContainExtensionMethods)
-                    {
-                        var moduleSymbol = _containingType.ContainingPEModule;
-                        isExtensionMethod = moduleSymbol.Module.HasExtensionAttribute(_handle, ignoreCase: false);
```

Comment:
This check is already done by `ContainingType.MightContainExtensionMethods`:

https://github.com/dotnet/roslyn/blob/99c1675c764c354a93e0d50ad53158494fcdfddb/src/Compilers/CSharp/Portable/Symbols/Metadata/PE/PENamedTypeSymbol.cs#L1729-L1731

Reply:
This is definitely wrong. I was just blind.